pkg/generated/objects/management.cattle.io/v3/objects.go

```diff
@@ -220,6 +220,59 @@ func FleetWorkspaceFromRequest(request *admissionv1.AdmissionRequest) (*v3.Fleet
 	return object, nil
 }
 
+// PodSecurityAdmissionConfigurationTemplateOldAndNewFromRequest gets the old and new PodSecurityAdmissionConfigurationTemplate objects, respectively, from the webhook request.
+// If the request is a Delete operation, then the new object is the zero value for PodSecurityAdmissionConfigurationTemplate.
+// Similarly, if the request is a Create operation, then the old object is the zero value for PodSecurityAdmissionConfigurationTemplate.
+func PodSecurityAdmissionConfigurationTemplateOldAndNewFromRequest(request *admissionv1.AdmissionRequest) (*v3.PodSecurityAdmissionConfigurationTemplate, *v3.PodSecurityAdmissionConfigurationTemplate, error) {
```

Comment:
I'm not seeing usage of this (or next) func. Am I missing implementation for validation/mutation for PSACTs?

Reply:
It is the next function not being used. 
This is a generated file, so we can not do much here.

Reply:
Ugh, my bad about generated code. 😞